client/sites-dashboard/components/sites-container.tsx

```diff
@@ -0,0 +1,139 @@
+import { ClassNames } from '@emotion/react';
+import styled from '@emotion/styled';
+import { createInterpolateElement } from '@wordpress/element';
+import { useI18n } from '@wordpress/react-i18n';
+import EmptyContent from 'calypso/components/empty-content';
+import { SiteData } from 'calypso/state/ui/selectors/site-data';
+import { SitesTable } from './sites-table';
+
+const EmptyContainer = styled.div`
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	height: 600px;
+
+	.empty-content__line {
+		max-width: 650px;
+	}
+`;
+
+type SitesContainerProps = {
+	sites: SiteData[];
+	status: string;
+};
+
+export const SitesContainer = ( { sites, status }: SitesContainerProps ) => {
+	const { __ } = useI18n();
+	if ( sites.length > 0 ) {
+		return (
+			<ClassNames>
+				{ ( { css } ) => (
+					<SitesTable
+						className={ css`
+							margin-top: 32px;
+						` }
+						sites={ sites }
+					/>
+				) }
+			</ClassNames>
+		);
+	}
+
+	switch ( status ) {
```

Comment:
Though it's fine to use a `switch` statement in a render function in React, it's not a common pattern. People usually find `switch`es hard to read and they feel kinda off in this context. As we're already using an `if` above, let's try to have some consistency and keep using early-exit ifs (no `else`s too).

But don't worry, there's nothing wrong with it, it's just that it's not common to see and a bit hard to read.

Reply:
yeah switches are not my favourite conditionals. Lets see if the ifs will read better